web/statik/handler.go

```diff
@@ -52,6 +52,7 @@ var (
 		"passphrase_reset.html",
 		"sharing_discovery.html",
 		"twofactor.html",
+		"new_app_available.html",
```

Comment:
Can you keep the list sorted please?

And I would like to have the new template added to https://github.com/cozy/cozy-stack/blob/master/docs/assets.md#html-pages. It is helpful when working on a custom theme.